src/spans/awsSpan.ts

```diff
@@ -50,6 +50,7 @@ export const MSSQL_SPAN = 'msSql';
 export const MYSQL_SPAN = 'mySql';
 export const NEO4J_SPAN = 'neo4j';
 export const ENRICHMENT_SPAN = 'enrichment';
+export const PRISMA_SPAN = 'prisma';
```

Comment:
Don't forget: I want to see it e2e in Lumigo. Let's make sure that you'll not get "span_type not found"

Reply:
You mean actual a test added to the _integration test_ repo with a Lambda running Prisma, or an E2E one-time "experiment" before we launch this?

Reply:
The first is preferred, but if it gets rough then let go with the second :)